pkg/vault/test/authorizer/TimelockAuthorizerMigrator.test.ts

```diff
@@ -42,6 +42,9 @@ describe('TimelockAuthorizerMigrator', () => {
     newAuthorizer = await deployedAt('TimelockAuthorizer', await migrator.newAuthorizer());
     const setAuthorizerActionId = await actionId(vault, 'setAuthorizer');
     await oldAuthorizer.grantRolesToMany([setAuthorizerActionId], [migrator.address]);
+
+    const CHANGE_ROOT_DELAY = await newAuthorizer.getRootTransferDelay();
+    await advanceTime(CHANGE_ROOT_DELAY);
```

Comment:
This reflects how nothing can be done until the delay passes, which will be annoying in the real world.

Reply:
We *could* move this down further but it's just here for simplicity.